drive/include/drive_configuration.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+
+namespace sjsu::drive {
+    /**
+     * @brief This structure defines a valid steering configuration for the drive system.

```

Comment:
document use of the variables better. Leave a brief for each variable inside the struct.